src/Sylius/Behat/Resources/config/services/contexts/setup.xml

```diff
@@ -120,6 +120,7 @@
             <argument type="service" id="sylius.order_item_quantity_modifier" />
             <argument type="service" id="doctrine.orm.entity_manager" />
             <argument type="service" id="Sylius\Calendar\Provider\DateTimeProviderInterface" />
+            <argument type="service" id="sylius.shop_user.token_generator.email_verification" />
```

Comment:
This ain't right, also the one in `Sylius\Bundle\ApiBundle\CommandHandler\Cart\PickupCartHandler` isn't correctly used either since there's no uniqueness checking.

Reply:
This will be implemented in another feature pull request

Reply:
Unresolved just so it'll be visible until fixed. 💃